plugins/engines/ospray/render/OSPRayScene.cpp

```diff
@@ -128,6 +128,29 @@ void OSPRayScene::_saveCacheFile()
     file.write( ( char* )&nbMaterials, sizeof( size_t ));
     BRAYNS_INFO << nbMaterials << " materials" << std::endl;
 
+    // Save materials
+    for( size_t materialId = 0; materialId < nbMaterials; ++materialId )
```

Comment:
C++11 range for over _materials won't work? Or nbMaterials != _materials.size()?